src/server.ts

```diff
@@ -364,6 +366,18 @@ export interface ServerOptions<E = unknown> {
     ctx: Context<E>,
     message: CompleteMessage,
   ) => Promise<void> | void;
+  /**
+   * An optional override for the JSON.parse function used to hydrate
+   * incoming messages to this server. Useful for parsing custom datatypes
+   * out of the incoming JSON.
+   */
+  reviver?: JSONReviver;
+  /**
+   * An optional override for the JSON.stringify function used to serialize
+   * outgoing messages to from server. Useful for serializing custom
+   * datatypes out to the client.
+   */
+  replacer?: JSONReplacer;
```

Comment:
Same as with the client-side; I prefer longer, but more descriptive names. Especially for options which can grow:
```suggestion
  jsonMessageReviver?: JSONMessageReviver;
  /**
   * An optional override for the JSON.stringify function used to serialize
   * outgoing messages to from server. Useful for serializing custom
   * datatypes out to the client.
   */
  jsonMessageReplacer?: JSONMessageReplacer;
```